doc/api/best-practices.md

```diff
@@ -0,0 +1,47 @@
+# Best Practices
+
+## Query Parameter Naming
+
+### Date from and to
+
+On APIs best practice is to use the parameter names ***dateFrom*** and ***dateTo***
+each parameter shall take its parameter as a **UNIX timestamp** in **UTC**.
+
+```text
+?dateFrom=1640991600
+&dateTo=1643670000
+```
+
+### Aggregation
```

Comment:
This sections seems out of place within best practices. It does not seem like a practice to me but rather a shared property.